README.md

```diff
@@ -1,30 +1,30 @@
-# Lance: A Columnar Data Format for Computer Vision
+# Lance: Deep Learning with DuckDB and Arrow
 
-![CI](https://github.com/eto-ai/lance/actions/workflows/cpp.yml/badge.svg)
-[![Docs](https://img.shields.io/badge/docs-passing-brightgreen)](https://eto-ai.github.io/lance/)
+<center>SELECT predict(‘resnet’, image) FROM dataset</center>
```

Comment:
should we use code block for this line, so that we can have highlights? Maybe we will lose center position tho.

Not a blocker

Reply:
We lose the center position
perhaps we use this for now, then use an image then we get the best of both worlds -- code format / center alignment

Reply:
👌